chaoslib/pumba/pod_failure_by_sigkill.yaml

```diff
@@ -81,9 +81,17 @@
 
       - name: Record the app_container
         set_fact:
-          app_container: "{{ container.stdout }}"
+          a_container: "{{ container.stdout }}"
```

Comment:
is `app_container` variable is already used, If not can you replace this

Reply:
We are passing the app_container variable as var in this file. As the priority of var is higher than the priority of set_fact so it will create a problem if we will pass a null string inside var. so we can't use same name for set_fact and var.

Reply:
okay!!